packages/html/src/treeview/templates/treeview-checkboxes.tsx

```diff
@@ -0,0 +1,28 @@
+import { Treeview, TreeviewItem, TreeviewGroup } from "../../treeview";
+
+export const TreeviewCheckboxes = (props) => (
+    <Treeview
+        children={[
+            <TreeviewItem text="Root 1" showCheckbox />,
```

Comment:
```suggestion
            <TreeviewItem text="Root 1" showCheckbox />
```

Reply:
The _TreeviewItem_ are defined as children, so I think we need the comma